fiasko_bro/tokenized_validators/tokenized_validators.py

```diff
@@ -6,7 +6,12 @@ def run_if_tokens_satisfy_condition(tokens, condition):
     def validator_decorator(func):
         @wraps(func)
         def func_wrapper(*args, **kwargs):
-            repo_tokens = kwargs.get('validator_token') or kwargs.get('validator_tokens')
+            repo_token = kwargs.get('validator_token')
+            if repo_token is not None:
+                repo_token = [repo_token]
+            repo_tokens = repo_token or kwargs.get('validator_tokens')
```

Comment:
This looks like a fine solution to me, but let's rewrite the line as `repo_tokens = repo_token or kwargs['validator_tokens']`. That way, we will be getting an exception about the missing argument early.